core/vm/contracts.go

```diff
@@ -136,6 +140,23 @@ func init() {
 	for k := range PrecompiledContractsBerlin {
 		PrecompiledAddressesBerlin = append(PrecompiledAddressesBerlin, k)
 	}
+	for k := range PrecompiledContractsBLS {
+		PrecompiledAddressesBLS = append(PrecompiledAddressesBLS, k)
+	}
+
+	// Set of all native precompile addresses that are in use
+	PrecompileAllNativeAddresses = make(map[common.Address]struct{})
+	for _, k := range append(PrecompiledAddressesBerlin, PrecompiledAddressesBLS...) {
```

Comment:
Why not include all previous hard fork native addresses here?

Reply:
This is the full set since `PrecompiledAddressesBerlin` includes all of the previous ones, but we can switch it anyways to make it more clearly correct.